src/main/java/reactor/netty/resources/ConnectionProvider.java

```diff
@@ -422,16 +437,25 @@ public ConnectionProvider build() {
 		Duration maxLifeTime;
 		boolean  metricsEnabled;
 		Function<PoolBuilder<PooledConnectionProvider.PooledConnection, ?>,
-				InstrumentedPool<PooledConnectionProvider.PooledConnection>> leasingStrategy = PoolBuilder::fifo;
+				InstrumentedPool<PooledConnectionProvider.PooledConnection>> leasingStrategy;
 
 		/**
 		 * Returns {@link ConnectionPoolSpec} new instance with default properties.
 		 */
 		private ConnectionPoolSpec() {
+			if (DEFAULT_POOL_MAX_IDLE_TIME != null) {
+				maxIdleTime(Duration.ofMillis(Long.parseLong(DEFAULT_POOL_MAX_IDLE_TIME)));
+			}
+			if("lifo".equals(DEFAULT_POOL_LEASING_STRATEGY)) {
```

Comment:
maybe extract that as a public constant in `ReactorNetty`? at least as a private constant next to `DEFAULT_POOL_LEASING_STRATEGY` above.

Reply:
done